Packs/Endace/Integrations/Endace/license.txt

```diff
@@ -0,0 +1,16 @@
+Copyright 2020 Endace Technology Limited

```

Comment:
not sure why this file is needed by please rename it to have the integration filename as prefix with `_` as separator, like other files in the dir (except README)

Reply:
Our company wants license file to be in all directories. I have to have in there. I have renamed them as Endace_license.txt